lib/php/rendition-factory/src/Transformer/Document/PdfToImageTransformerModule.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Alchemy\RenditionFactory\Transformer\Document;
+
+use Alchemy\RenditionFactory\DTO\FamilyEnum;
+use Alchemy\RenditionFactory\DTO\OutputFile;
+use Alchemy\RenditionFactory\DTO\InputFileInterface;
+use Alchemy\RenditionFactory\DTO\OutputFileInterface;
+use Alchemy\RenditionFactory\Transformer\TransformationContext;
+use Alchemy\RenditionFactory\Transformer\TransformerModuleInterface;
+use Spatie\PdfToImage\Enums\OutputFormat;
+use Spatie\PdfToImage\Pdf;
+
+final readonly class PdfToImageTransformerModule implements TransformerModuleInterface
+{
+    public static function getName(): string
+    {
+        return 'pdf_to_image';
+    }
+
+    public function transform(InputFileInterface $inputFile, array $options, TransformationContext $context): OutputFileInterface
+    {
+        if ($inputFile->getType()!=='application/pdf') {
+            throw new \InvalidArgumentException('Invalid input file');
```

Comment:
Can you please add this TODO comment? We'll do it later:

```suggestion
            // TODO normalize file to PDF
            throw new \InvalidArgumentException('Invalid input file');
```